lib/contracts/role-user-community.ts

```diff
@@ -147,13 +147,23 @@ export const roleUserCommunity: RoleContractDefinition = {
 				},
 				{
 					properties: {
-						id: {
```

Comment:
If I'm reading the old role schema correctly, it seems like it should have never matched anything? Trying to match `authentication` or `user-settings` contracts with the same `id` as the `user`...

Reply:
@joshbwlng yes lol. No idea where my head was at at the time.......